rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ProjectSpecTest.java

```diff
@@ -476,4 +476,10 @@ public void should_document_create_project() throws Exception {
                                 fieldWithPath("_embedded.createdBy").description("The user who created this project")
                         )));
     }
+
+    @Test
+    public void should_document_upload_attachment_to_project() throws Exception {
+    testAttachmentUpload("/api/projects/", project.getId());
```

Comment:
Please use proper indents in these lines

Reply:
Done.